File: src/main/java/com/partymakers/shareparty/application/party/dto/Expense.java
package com.partymakers.shareparty.application.party.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Data;

@Data
public class Expense {
    @JsonProperty("name")
    private final String  name;
    @JsonProperty("cost")
    private final Integer cost;
    @JsonProperty("count")
    private final Double count;
}

File: src/main/kotlin/com/partymakers/shareparty/application/party/dto/Expense.kt
package com.partymakers.shareparty.application.party.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class Expense(
    @JsonProperty("name")
    val name: String,
    @JsonProperty("cost")
    val cost: Int,
    @JsonProperty("count")
    val count: Double
) 